ppr-api/src/ppr_api/resources/drafts.py

```diff
@@ -17,7 +17,7 @@
 
 from http import HTTPStatus
 
-from flask import request, jsonify
+from flask import request, jsonify, g
```

Comment:
doesn't the linter complain about alphabetical order here?

Reply:
No complaints but sorted anyway as a second commit.